src/test/regress/expected/pg16.out

```diff
@@ -545,3 +545,16 @@ SET search_path TO pg16;
 \set VERBOSITY terse
 SET client_min_messages TO ERROR;
 DROP SCHEMA pg16 CASCADE;
+
```

Comment:
This pg16.out file is not corresponding to the current pg16.sql. Could you please update it?